### DISCUSSION TITLE
macOS Sonoma 14.0

### DISCUSSION THREAD

**Discussion by dmtroyer** [abiosoft/colima#820] [General]
Have people had luck running colima v0.5.5 on macOS Sonoma 14.0? I'm on a 14" M1 MBP. I would be very sad to break my colima :-)

**Comment by andressltz** [comment]
Success here. MBP 14" M1 PRO. Before upgrade, I create a new volume and use dualboot, in the main volume macOS Ventura and in the new volume I run macOS Sonoma Beta, so I tested for a time before upgrade the main volume.

**Comment by ahmed-adly-khalil** [comment]
success here, colima is amazing !!! 
my m1 mb pro was dying with docker desktop, now it runs like nothing is installed!!

Thank you @abiosoft !!
